dca632fffe7697e0.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "dca632fffe7697e0",
+    "servers": [
+      {
+        "server_address": "router.eu.thethings.network",
```

Comment:
BTW, if you're in Switzerland, you might want to consider using the swiss backend: `ttn.opennetworkinfrastructure.org` (https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses).